test_suite.py

```diff
@@ -55,16 +69,21 @@ def docker_image(docker_client: docker.DockerClient, once_per_test_suite_run) ->
 class DockerRunner:
     client: docker.DockerClient
     image: str
+    language: str
```

Comment:
This type annotation is wrong now, isn't it?

I'm not sure yet what the right annotation would be. My first thought is that if we made a class for the language classes to inherit from, it would be that.

E.g. something like this:

```python
@dataclass
class Language:
  name: str
  def script(self, test_name):
    raise "Not Implemented"

class Python(Language):
  name = "python"
  ...

class DockerRunner:
  ...
  language: Language
```

Reply:
Yeah, that's a good call. We can also make an abstract base class `Language` to inherit from.